client/models/user.coffee

```diff
@@ -48,10 +48,14 @@ module.exports = class User extends Backbone.Model
       user.partner_ids = jwtDecode(@get('access_token')).partner_ids
       user.channel_ids = _.pluck results[1], 'id'
 
+
       for attr in ['id', 'type', 'name', 'email']
         return callback true unless _.isEqual user[attr], @get(attr)
       for attr in ['channel_ids', 'partner_ids']
-        return callback true unless _.isEqual(user[attr].sort(), @get(attr).sort())
+        return callback true unless _.isEqual(
+          _.clone(user[attr]).sort(),
+          _.clone(@get(attr)).sort()
+        )
```

Comment:
Seemed to just happen locally, but we were getting an error that sorting wasn't allowed on these read-only properties: 

![image](https://user-images.githubusercontent.com/2236794/37310812-8950e4e8-261b-11e8-95a4-a215675e56ea.png)

Reply:
👍